config/settings.yml

```diff
@@ -22,6 +22,8 @@ shared:
     virtual_appliance: 'https://ontoportal.github.io/documentation/administration'
     widgets: 'http://www.bioontology.org/wiki/NCBO_Widgets'
     wiki: 'https://www.bioontology.org/wiki/'
+    mappings: 'https://www.bioontology.org/wiki/BioPortal_Help#Mappings_Tab'
+    metadata_help: 'https://doc.jonquetlab.lirmm.fr/share/32c082ff-97f5-45f4-bafe-05f41c9a5ce4'
```

Comment:
metadata_help points to a page that redirects to industryportal's submission-guide.  
Is there a plan to make this more generic?